lib/foreman/renderer/scope/macros/host_template.rb

```diff
@@ -121,6 +121,29 @@ def ks_console
             (@port && @baud) ? "console=ttyS#{@port},#{@baud}" : ''
           end
 
+          apipie :method, 'Returns manage_packages snippet with packages & action' do
+            required :action, String, "Allowed actions: 'install', 'update', 'remove', 'group install' and 'group remove'"
+            required :packages, Array, 'Packages to manage'
```

Comment:
It accepts strings too. Can apipie describe that?

Reply:
> Can apipie describe that?
I checked docs but could not find how to define multiple types to one parameter.

Reply:
Welp, that's probably my bad :(

`apipie` for describing methods should almost fully compatible with `api` for describing endpoints.

Thus, if a parameter can accept string or array try to use
`required :packages, [Array, String], 'Packages to manage'`